smoketest.sh

```diff
@@ -303,6 +310,7 @@ createPod() {
         --publish 8081:8081 \
         --publish 8082:8082 \
         --publish 8083:8083 \
+        --publish 9091:9091 \
```

Comment:
do we need this still?

Reply:
Probably not. I think if you wanted to run this script and then connect a JMX client from outside of the pod (ex. JMC as a process directly on the host) then this would be needed. I'll try that out.

Reply:
That seems to just hang JMC... :/